File: app/src/main/java/com/stedi/multitouchpaint/dialogs/ExitDialog.java
package com.stedi.multitouchpaint.dialogs;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.stedi.multitouchpaint.App;
import com.stedi.multitouchpaint.R;

import butterknife.OnClick;

public class ExitDialog extends BaseDialog {
    public class Callback {
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return butterKnifeInflate(inflater, R.layout.exit_dialog, container);
    }

    @OnClick({R.id.done, R.id.cancel})
    public void onButtonsClick(View v) {
        if (v.getId() == R.id.done)
            App.getBus().post(new Callback());
        dismiss();
    }
}

File: app/src/main/java/com/stedi/multitouchpaint/dialogs/ExitDialog.kt
package com.stedi.multitouchpaint.dialogs

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import butterknife.OnClick
import com.stedi.multitouchpaint.App
import com.stedi.multitouchpaint.R

class ExitDialog : BaseDialog() {

    class Callback

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        return butterKnifeInflate(inflater, R.layout.exit_dialog, container)
    }

    @OnClick(R.id.done, R.id.cancel)
    fun onButtonsClick(v: View) {
        if (v.id == R.id.done) {
            App.BUS.post(Callback())
        }
        dismiss()
    }
}